server/shared/common.js

```diff
@@ -1123,3 +1125,19 @@ export const TermTypeGroups = {
 	SNP_LIST: 'SNP List',
 	SNP_LOCUS: 'SNP Locus'
 }
+
+export function isNumeric(term) {
```

Comment:
@airenzp please change function name to `termIsNumeric()` indicate that the function works on terms